docs/features/Rate-reminder.md

```diff
@@ -36,3 +36,121 @@ You can find examples on how to implement **Rate Reminder** for the following pl
 
 * [iOS](../../docs/guides/iOS/ios-rate-reminder.html)
 * [Android](../../docs/guides/Android/android-rate-reminder.html)
+
+
+# Version 2
+
+Ratereminders V2 has the following endpoints:
+
+ - [GET] Show
+ - [GET] Actions
+ - [Post] Action Events
+ - [Post] Answer
+
+# GUID
```

Comment:
We have a section here also
https://github.com/nstack-io/documentation/blob/master/docs/API.md#what-is-a-guid

maybe just point to that? and then write a bit more about what your mean by "storing your id" and what is a GUID

Reply:
Good point. I will look into it now.

Reply:
@Casperhr  i have linked to GUID. I'm not sure if i think that the guid part should be removed from rate-reminder. How do you like the current version ? :)